src/developer/web-api/analytics.md

```diff
@@ -2816,6 +2816,310 @@ Table: Query parameters
 | svg | Yes | The SVG content |
 | filename | No | The file name for the returned attachment without file extension |
 
+## Analytics outlier detection { #webapi_analytics_outlier_detection } 
+
+The analytics API provides endpoints for investigation of the data quality based on Z Score and Modified Z Score. Both scores are statistical measures that help analyze and interpret data in the context of deviations from the middle value. They are particularly useful in identifying outliers or extreme values in a dataset. The API is implemented as a single analytics endpoint:
```

Comment:
I believe that we should add the "outlier" word to the sentence:
"The analytics **outlier** API provides"

Reply:
done